apps/labs/posts/sympy_documentation.md

```diff
@@ -0,0 +1,356 @@
+The three main takeaways of the survey were:
+
+1. The main SymPy documentation site (https://docs.sympy.org) is overwhelmingly
+  the most popular resource that people use to get help with SymPy. This is
+  true across all levels of experience, compared to other resources like the
+  SymPy website, StackOverflow, and community sites (note: this survey was
+  given in 2021, before the popularity of LLM tools like ChatGPT, so this was
+  not included as an option for respondents).
+  Consequently, we decided that it would be most impactful to spend efforts on
+  improving the documentation site over those other resources.
+
+  ![Plot showing results of SymPy documentation question "which of the
+  following resources do you use when getting help with SymPy (select all that
+  apply)?" The choice "SymPy Docs Website" has the most responses at
+  91%.](/posts/sympy-documentation/sympy-survey-results.svg)
+
+2. Survey respondents identified many deficiencies in the SymPy docs which
+   made it clear that certain improvements needed to be made to the overall
+   layout and organization of SymPy documentation site. In particular, we
+   identified 4 major improvements that could be made:
+
+   - Better top-level organization.
+   - A better Sphinx theme that provides better sidebar navigation.
+   - The docs have many large pages which would benefit from being split into smaller pages.
+   - There were several issues with the SymPy Live extension. This was a
+     Sphinx extension that allowed users to execute the example code blocks in
+     the SymPy documentation directly in their browser.
+
+   Of these, all except the third, splitting large pages, were done as part of
+   the CZI grant work. See below for more details. Splitting large pages
+   hasn't been done yet due to technical difficulties with the Sphinx autodoc
+   extension, as well as due to the fact that the large pages are now much
+   easier to navigate with the new Sphinx theme (see below).
+
+   The SymPy Live extension in the documentation was removed, as it was
+   considered too much of a maintenance burden for the SymPy community to
+   maintain. There is a new [SymPy Live](https://live.sympy.org/) shell that
+   runs on [JupyterLite](https://jupyter.org/try-jupyter/lab/index.html)—that
+   is, it runs entirely in the browser using
+   [Pyodide](https://pyodide.org/en/stable/). We are hopeful that the
+   JupyterLite community can come up with an equivalent SymPy Live-like
+   extension that so that we can re-enable similar functionality in the SymPy
+   documentation.
+
+3. We were been able to identify some primary areas of documentation that
+   to prioritize for writing new documentation guides.
+
+Click if you wish to read the [full survey results](https://www.sympy.org/sympy-docs-survey/2021-docs-survey.html).
+
+# Improved Sphinx Theme - Furo
+
+<!-- TODO: organize these pictures so that the text is a caption -->
```

Comment:
These are actually parsed as MDX, so comments are spelled:

```suggestion
{/* TODO: organize these pictures so that the text is a caption */}
```

Reply:
We need to just address these comments before merging anyway. Do you know if there's a good way to do this? Otherwise, let's just leave them as images like they are now.